site/ui/components/shared/Navbar.vue

```diff
@@ -126,6 +126,7 @@ export default {
       }
       return [];
     },
+    staticPages: (Array) => ['About', 'Contact', 'DatasetNotFound'],
```

Comment:
Just so you know, this is actually a function that returns an array, rather than just being an array. I'll fix it though!

Reply:
All I ended up doing was declaring a property in `data` rather than a getter function in `computed`. This way it doesn't have to call a function every time to get the list of `staticPages`.